stable/aws-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,24 @@
+tls_secret_name: pod-identity-webhook
+annotation_prefix: eks.amazonaws.com
+token_audience: sts.amazonaws.com
+
+replicas: 3
+
+image:
+  repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pod-identity-webhook
+  tag: latest
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
```

Comment:
Also, for other charts, we've just used a generic `podAnnotations` value:

https://github.com/aws/eks-charts/blob/master/stable/aws-vpc-cni/values.yaml#L29

https://github.com/aws/eks-charts/blob/575321fdf9d68a656c75768805cc80175b92275e/stable/aws-vpc-cni/templates/daemonset.yaml#L17-L21

Maybe do that for this chart as well?

Reply:
OK serviceAccount thing is done.

`podAnnotations` thing is done.

RBAC is not optional any more I believe and I don't know how to write a PSP so perhaps this can follow in a later PR?

Reply:
Yup, PSP stuff can ceertainly be done in followup